spec/factories/factories.rb

```diff
@@ -231,6 +231,13 @@
     name  "No"
   end
 
+  factory :sign_up_topic, class: SignUpTopic do
```

Comment:
In `spec/factories/factories.rb`, there has already been a factory named `topic`. Why do you add another one?